dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
I found an interesting thing - https://github.com/jrburke/requirejs/wiki/Updating-existing-libraries#anon

If I understand correctly, if `ckeditor.js` was concatenated with other scripts (e.g. by an assets pipeline like in RoR) it may lead to an error if we use an anonymous module definition (which we do now).

This means that there must be (somehow available) a totally non-AMD version of `ckeditor.js` available for situations like this - concatenation by some simple tool. Therefore at least the builder must have an option for that.

The remaining questions are:

1. What options the builder should have?
2. What do we want to ship in a release?

Ad 1. I think that 3 - AMD with polluting (could be the default), AMD without polluting (for pedants like many of us), no AMD at all.

Ad 2. jQuery is doing the "AMD with polluting", so I think we can do it as well.